rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMClientExample.java

```diff
@@ -0,0 +1,23 @@
+package org.ow2.proactive_grid_cloud_portal.rm.client;
+
+import org.objectweb.proactive.ActiveObjectCreationException;
+import org.objectweb.proactive.core.node.NodeException;
+import org.ow2.proactive.resourcemanager.exception.RMException;
+import org.ow2.proactive_grid_cloud_portal.common.RMRestInterface;
+
+import javax.security.auth.login.LoginException;
+import java.security.KeyException;
+
+public class RMClientExample {
+
+    public static void main(String[] args) throws LoginException, KeyException, ActiveObjectCreationException, NodeException, RMException {
+        System.out.println("Hello from example.");
+
+//        HttpClientBuilder httpClientBuilder = HttpClientBuilder.create().useSystemProperties();
+        RMRestClient client = new RMRestClient("http://localhost:8080/rest/", null);//, new ApacheHttpClient4Engine(httpClientBuilder.build()));
+        RMRestInterface rm = client.getRm();
+        String sessionId = rm.rmConnect("admin", "admin");
```

Comment:
This is ok as a simple proxy (and in the way the SchedulerRestClient is made).

To be elegantly usable in a task like the schedulerapi, a second proxy should be made.

For example, the schedulerapi binding can be used like this inside a task:

schedulerapi.connect() (automatically uses the scheduler url transfered to the task by variable, and the current user credentials)
schedulerapi.submitJob(myJob)  (schedulerapi implements the scheduler interface).

This proxy is created in the class org.ow2.proactive.scheduler.task.client.SchedulerNodeClient